### ISSUE TITLE
Open source license

### ISSUE THREAD

**Issue by edwardrjones97** [jhnnsrs/turms#61]
Hi, `turms` looks awesome and I was wondering if it was intentionally missing open source licensing information?

PyPI currently reports:
```
License: Other/Proprietary License (CC BY-NC 3.0)
```
and I'd expect to see a license file somewhere in this repository specifying the applicable license.

**Comment by jhnnsrs** [comment]
Good point! CC 3.0 was chosen at a time where i was unsure of where the project  will head, but i am happy to change it to something more permissive now. Guess BSD or MIT? Not sure about any of the legal implications though :D Happy for suggestions?
